test/unit/test-custom-element-v1.js

```diff
@@ -140,6 +140,41 @@ describes.realWin('CustomElement V1', {amp: true}, (env) => {
       doc.body.appendChild(element);
       expect(element.readyState).to.equal('building');
     });
+
+    it('should reschedule build when re-attached after build', async () => {
+      const element = new ElementClass();
+
+      builderMock.expects('schedule').withExactArgs(element).twice();
+      builderMock.expects('unschedule').withExactArgs(element).once();
+
+      doc.body.appendChild(element);
+      await element.buildInternal();
```

Comment:
Nit: can we start with an `expect` for readyState to be BUILDING?

Reply:
Good idea. Done.